### ISSUE TITLE
statistics not working  for me

### ISSUE THREAD

**Issue by marcosmaia** [crocodic-studio/crudbooster#1037]
select date(created_at) AS label, value_total AS value, currency AS label  from withdrawn  where month(created_at) = MONTH(CURDATE())

does not display label currency, 
display only 2 values date and total.
How do I display 3? date, total, currency

I will be very grateful for the help

**Comment by fdigennaro** [comment]
Create a view in your database and then use this for statistics.

`CREATE VIEW view_last_month AS 
     SELECT DATE(created_at) AS label, value_total AS value, currency AS label FROM withdrawn WHERE MONTH(created_at) = MONTH(CURDATE())
`

In statistics module put 

`SELECT * FROM view_last_month`

**Comment by marcosmaia** [comment]
I'll try, thanks for the help.

**Comment by marcosmaia** [comment]
Mensagens do MySQL : 
error Duplicate label column name

CREATE VIEW view_last_month AS SELECT DATE(created_at) AS label, value_total AS value, currency AS label FROM withdrawn WHERE MONTH(created_at) = MONTH(CURDATE())

**Comment by fdigennaro** [comment]
Sorry, two column with same label "label", change one

**Comment by marcosmaia** [comment]
I changed it to date anyway so only label and value appears

CREATE VIEW view_last_month AS SELECT DATE(created_at) AS date, value_total AS value, currency AS label FROM withdrawn WHERE MONTH(created_at) = MONTH(CURDATE())

**Comment by fdigennaro** [comment]
Have you created the view in your database or are you inserting the SQL string in the crudbooster statistics module?

**Comment by marcosmaia** [comment]
yes create and in the module I put SELECT * FROM view_last_month

**Comment by jdredondo** [comment]
try this:
SELECT DATE(created_at) AS label, value_total AS value FROM withdrawn WHERE MONTH(created_at) = MONTH(CURDATE());
SELECT value_total AS value, currency AS label FROM withdrawn WHERE MONTH(created_at) = MONTH(CURDATE())
